website/common/script/ops/feed.js

```diff
@@ -68,9 +70,28 @@ export default function feed (user, req = {}, analytics) {
     throw new NotAuthorized(i18n.t('messageAlreadyMount', req.language));
   }
 
+  if (!Number.isInteger(amount) || amount < 0) {
+    throw new BadRequest(i18n.t('invalidAmount', req.language));
```

Comment:
the tests are failing because invalidAmount doesn't exist as a string in ` website/common/locales/en/pets.json`